test/Graphics/CircleShape.test.cpp

```diff
@@ -108,4 +108,15 @@ TEST_CASE("[Graphics] sf::CircleShape")
         CHECK(triangle.getPoint(1) == Approx(sf::Vector2f(3.732050896f, 3.000000000f)));
         CHECK(triangle.getPoint(2) == Approx(sf::Vector2f(0.267949224f, 3.000000000f)));
     }
+
+    SUBCASE("Geometric center")
+    {
+        CHECK(sf::CircleShape(2.f).getGeometricCenter() == sf::Vector2f(2.f, 2.f));
+    }
+
+    SUBCASE("Geometric center for three points")
+    {
+        sf::CircleShape circle(4.f, 3);
+        CHECK(circle.getGeometricCenter() == (circle.getPoint(0) + circle.getPoint(1) +  circle.getPoint(2)) / 3.f);
```

Comment:
Instead of replicating the implementation which creates a tautological test, will you please directly construct a vector with the correct value?